src/Microsoft.ML.StandardLearners/Standard/SdcaRegression.cs

```diff
@@ -25,6 +25,12 @@
 namespace Microsoft.ML.Runtime.Learners
 {
     /// <include file='doc.xml' path='doc/members/member[@name="SDCA"]/*' />
+    /// <example>
+    /// <format type="text/markdown">
+    /// <![CDATA[
+    ///  [!code-csharp[SDCA Regression](../../../docs/samples/Microsoft.ML.Samples.StaticPipe/Trainers.cs?range=5-8,12-14,25-79) "The SDCA regression example."]
+    /// ]]></format>
+    /// </example>
```

Comment:
@dend, @mairaw does the path look correct? In the examples it says it should be relative to the file containing the XML; but when I configured the CRR in the docs, there is a  

"path_to_root": "docs/samples"

property. Does the path need to be relative to that?

Reply:
I don't know @sfilipi. The way we use it in our docs is a bit different. Tagging @JRAlexander to help investigate. 
We also do use the notation of ~ to start at the root as well.